README.md

```diff
@@ -1 +1,966 @@
+jms:Session jmsSession2 = new(conn, {
+        // Optional property. Defaults to AUTO_ACKNOWLEDGE
+        acknowledgementMode: "AUTO_ACKNOWLEDGE"
+    });
+
+// Initialize a queue receiver using the created session
+endpoint jms:QueueReceiver jmsConsumer2 {
+    session:jmsSession2,
+    queueName:"DeliveryQueue"
+};
+service<jms:Consumer> deliverySystem bind jmsConsumer2 {
+    // Triggered whenever an order is added to the 'OrderQueue'
+    onMessage(endpoint consumer, jms:Message message2) {
+        log:printInfo("New order successfilly received from the Delivery Queue");
+        // Retrieve the string payload using native function
+        string stringPayload2 = check message2.getTextMessageContent();
+        log:printInfo("Delivery Details: " + stringPayload2);
+
+        log:printInfo(" Delivery Details sent to the customer successfully");
+    }
+}
+```
+
+
+## Testing 
+
+### Invoking the service
+
+- First, start the `Apache ActiveMQ` server by entering the following command in a terminal from `<ActiveMQ_BIN_DIRECTORY>`.
+
+```bash
+   $ ./activemq start
+```
+
+- Navigate to `message_construction_patterns/guide` and run the following commands in separate terminals to start both `phone_store_service` and `order_delivery_service`.
+```bash
+   $ ballerina run phone_store_service.bal
+```
+
+```bash
+   $ ballerina run order_delivery_system
+```
+   
+- Invoke the `phone_store_service` by sending a GET request to check the available books.
+
+```bash
+   curl -v -X GET localhost:9090/phonestore/getPhoneList
+```
+
+  The phone_store_service sends a response similar to the following.
+```
+   < HTTP/1.1 200 OK
+   ["Apple:190000","Samsung:150000","Nokia:80000","HTC:40000","Huawei:100000"]
+```
+   
+- Place an order using the following command.
+
+```bash
+   curl -v -X POST -d \
+   '{"Name":"John", "Address":"20, Palm Grove, Colombo, Sri Lanka", 
+   "ContactNumber":"+94718930874", "PhoneName":"Apple:190000"}' \
+   "http://localhost:9090/phonestore/placeOrder" -H "Content-Type:application/json"
+   
+```
+
+  The bookstoreService sends a response similar to the following.
+```
+   < HTTP/1.1 200 OK
+   {"Message":"Your order is successfully placed. Ordered book will be delivered soon"} 
+```
+
+  Sample Log Messages:
+```bash
+
+  INFO  [phone_store_service] - order will be added to the order  Queue; CustomerName: 'Bob', OrderedPhone: 'Apple:190000'; 
+  INFO  [phone_store_service] - New order successfilly received from the Order Queue 
+  INFO  [phone_store_service] - Order Details: {"customerName":"John","address":"20, Palm Grove, Colombo, Sri Lanka","contactNumber":"+94718930874","orderedPhoneName":"Apple:190000"} 
+  
+  Order Details have sent to phone_order_delivery_service.
+
+
+  Order Details have received from phone_store_service
+  
+  INFO  [phone_order_delivery_service] - order Delivery details  added to the delivery  Queue; CustomerName: 'Bob', OrderedPhone: 'Apple:190000'; 
```

Comment:
Delivery -> delivery

Need to use cases consistently across the docs. Check in other places as well. Also, I think the 'Q' in Queue can be simple as well right?